presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -3251,115 +3252,96 @@ public void testShowCreateTable()
         assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), createTableSql);
     }
 
-    @Test
-    public void testCreateExternalTable()
+    /**
+     * Boilerplate wrapper for tests creating external textfile tables.
```

Comment:
We are rarely doing javadoc comments in the codebase as they tend to get out of sync with the code.
I would drop this one as method name and parameters are self-explanatory enough.